js/src/dropdown.js

```diff
@@ -14,6 +14,7 @@ import {
   isVisible,
   isRTL,
   noop,
+  toType,
```

Comment:
This is missing

Reply:
Can I add this to the export in util.js or should I perform the check manually?

Reply:
Just make sure the PR passes any tests locally before pushing :)

Reply:
Yeah I did this as a quick draft in the github web ui but have now cloned everything locally